docker/wforce_image/wforce.conf.j2

```diff
@@ -1,7 +1,9 @@
 -- This configuration file is provided as an example
 -- It only scratches the surface of what weakforced policy can do
 
-webserver("0.0.0.0:{{env['WFORCE_HTTP_PORT']}}", "{{env['WFORCE_HTTP_PASSWORD']}}")
+addListener("0.0.0.0:{{env['WFORCE_HTTP_PORT']}}", false,    "",             "",         {})
```

Comment:
all this whitespace seems pointless without a comment like in `trackalert.conf`

Reply:
Yes, indeed I will add the additional comments as per `trackalert.conf`